<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Текстовый редактор</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="boxActions">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="valHotKey">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setHotKey">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>110</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Назначить клавишу</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>201</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>961</width>
      <height>801</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QMdiArea" name="mdiArea">
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="tabsMovable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="CreateFile"/>
    <addaction name="OpenReadOnly"/>
    <addaction name="OpenFile"/>
    <addaction name="addFile"/>
    <addaction name="separator"/>
    <addaction name="Save"/>
    <addaction name="SaveFile"/>
    <addaction name="printText"/>
    <addaction name="separator"/>
    <addaction name="OpenRef"/>
    <addaction name="QuitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Язык</string>
     </property>
     <addaction name="separator"/>
     <addaction name="LangToRus"/>
     <addaction name="LangToEng"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Тема</string>
     </property>
     <addaction name="action_2"/>
     <addaction name="action_3"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="SaveFile">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="OpenRef">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="LangToRus">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="LangToEng">
   <property name="text">
    <string>Английский</string>
   </property>
  </action>
  <action name="OpenReadOnly">
   <property name="text">
    <string>Открыть для чтения</string>
   </property>
  </action>
  <action name="OpenFile">
   <property name="text">
    <string>Открыть для редактирования</string>
   </property>
  </action>
  <action name="addFile">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="QuitAction">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Светлая</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Тёмная</string>
   </property>
  </action>
  <action name="CreateFile">
   <property name="text">
    <string>Создать </string>
   </property>
  </action>
  <action name="printText">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
